src/pybind/mgr/dashboard_v2/module.py

```diff
@@ -74,6 +79,20 @@ def __init__(self, *args, **kwargs):
         mgr.init(self)
         self._url_prefix = ''
 
+    def can_run(self):
```

Comment:
the `can_run` method in `../mgr_module.py` is declared as a static method. As it is declared here breaks the module loading. Please make this a static method.

Reply:
Oops.  Updated.  Had to un-property-ize the frontend_path method to get it usable from here too.